tests/handlers/api/compute/test_vpcs.py

```diff
@@ -83,28 +84,19 @@ def test_vpcs_nio_update_udp(http_compute, vm):
                                     "rhost": "127.0.0.1",
                                     "filters": {}},
                                 example=True)
-    assert response.status == 201
+    assert response.status == 201, response.body.decode("utf-8")
```

Comment:
Adding the response body allow when the test fail to see the content of the body to debug